Ktisis/Interface/Windows/ConfigWindow.cs

```diff
@@ -94,6 +94,8 @@ private void DrawCategoriesTab() {
 		this.DrawHint("config.categories.hint_nsfw");
 		refresh |= ImGui.Checkbox(this.Locale.Translate("config.categories.show_all_viera_ears"), ref this.Config.Categories.ShowAllVieraEars);
 		this.DrawHint("config.categories.hint_viera_ears");
+		refresh |= ImGui.Checkbox(this.Locale.Translate("config.categories.show_friendly_bone_names"), ref this.Config.Categories.ShowFriendlyBoneNames);
+		this.DrawHint("config.categories.hint_friendly_bones");
```

Comment:
A refresh necessary here? It doesn't look like the bone names are cached.

```suggestion
		ImGui.Checkbox(this.Locale.Translate("config.categories.show_friendly_bone_names"), ref this.Config.Categories.ShowFriendlyBoneNames);
		this.DrawHint("config.categories.hint_friendly_bones");
```